buildAndReleaseTask/task.json

```diff
@@ -76,6 +76,15 @@
             "required": false,
             "helpMarkDown": "If you want Cobertura reports to be generated for the test execution, enable this.",
             "groupName": "codeCoverage"
+        },
+		{
+            "name": "enableAzure",
+            "type": "boolean",
+            "label": "Enable Azure SQL",
+            "defaultValue": "false",
```

Comment:
As we've been working on a cloud world, do you think that it makes sense to leave it true by default?